src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/getParams.tsx

```diff
@@ -113,32 +115,32 @@ const getUtcValue = (maybe: string | string[] | undefined | null): string | unde
     return maybe.find(needle => !!parseUtcValue(needle));
   }
 
-  maybe = parseUtcValue(maybe);
+  return parseUtcValue(maybe);
+};
 
-  if (typeof maybe === 'string') {
-    return maybe;
-  }
+type ParamValue = string | string[] | undefined | null;
 
-  return undefined;
+type ParsedParams = {
+  start?: string;
+  end?: string;
+  period?: string;
+  utc?: string;
+  [others: string]: string | null | undefined;
 };
 
-type ParamValue = string | string[] | undefined | null;
-type Params = {
-  start?: ParamValue;
-  end?: ParamValue;
+type InputParams = {
+  start?: Date | ParamValue;
+  end?: Date | ParamValue;
   period?: ParamValue;
   statsPeriod?: ParamValue;
-  utc?: ParamValue;
+  utc?: boolean | ParamValue;
+  [others: string]: any;
 };
-type RestParams = {[others: string]: ParamValue};
 
 export function getParams(
-  params: Params & RestParams,
+  params: InputParams,
```

Comment:
This was widened to allow Date instances to be passed in. The indexed type had to be widened as well unfortunately.